<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HW1_F74076019</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Contour">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>1. Find Contour</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_DrawContour">
       <property name="text">
        <string>1.1 Draw Contour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CountCoin">
       <property name="text">
        <string>1.2 Count Coin</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Coin1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>There are _ coins </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coin1_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>in coin01.jpg</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coin2_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>in coin02.jpg</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coin2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>There are _ coins </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_AR">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>3. Augmented Reality</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_AR">
       <property name="text">
        <string> 3.1 AR Pyramid</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Cali">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>161</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>2. Calibration</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>160</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_FindCor">
       <property name="text">
        <string>2.1 Find Corners</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_intr">
       <property name="text">
        <string>2.2 Find Intrinsic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
       <property name="maxCount">
        <number>15</number>
       </property>
       <item>
        <property name="text">
         <string>1.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15.bmp</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_extr">
       <property name="text">
        <string>2.3 Find Extrinsic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Dist">
       <property name="text">
        <string>2.4 Find Distortion </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_disparityMap">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>4.  Disparity Map	</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_DisMap">
       <property name="text">
        <string> 4. Disparity Map	</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Resnet">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>161</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>5. ResNet50</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_Train">
       <property name="text">
        <string>5.1 Start Training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_TensorB">
       <property name="text">
        <string>5.2 TensorBoard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_test">
       <property name="text">
        <string>5.3 Testing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_DataAug">
       <property name="text">
        <string>5.4 Data Augment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
